pom.xml

```diff
@@ -599,6 +599,38 @@ under the License.
 
 	<profiles>
 		<profile>
+			<id>scala-2.10</id>
+			<activation>
+				<property>
+					<!-- this is the default scala profile -->
+					<name>!scala-2.11.profile</name>
+				</property>
+			</activation>
+			<properties>
+				<!-- Scala / Akka -->
+				<scala.version>2.10.4</scala.version>
+				<scala.binary.version>2.10</scala.binary.version>
+				<scala.macros.version>2.0.1</scala.macros.version>
+				<akka.version>2.3.7</akka.version>
```

Comment:
The akka and scala macros version seems to be independent of the profile.
I would put them back to the `properties` section above.